cloud/main.js

```diff
@@ -357,6 +357,28 @@ function updateChatRoomOccupants(chatRoomId) {
     });
 }
 
+// Removes a User from a ChatRoom by deactivating the active Alias and
+// fires a pubnub presence event.
+// Takes: {aliasId: string, pubkey: string, subkey: string}
+// Returns: Alias
+Parse.Cloud.define("deleteChatEventForAlias", function (request, response) {
+    var requiredParams = ["aliasId","chatEventId"];
+    var params = request.params;
+    checkMissingParams(params, requiredParams, response);
+
+    Alias.get(params.aliasId).then(function(alias) {
+        var deletedChatEventIds = alias.get("deletedChatEventIds");
+        if (!deletedChatEventIds) {
+            deletedChatEventIds = [];
+        }
+        deletedChatEventIds = deletedChatEventIds.concat([params.chatEventId]);
+        alias.set("deletedChatEventIds", deletedChatEventIds);
+        alias.save().then(function(alias) {
```

Comment:
Oh wait nvm I'm dumb, try this:
`alias.save().then(response.success, response.error)`